machine-learning/ml/preprocessing/preprocessing.py

```diff
@@ -73,6 +73,11 @@ def retrieve_labelled_instances(dataset, refactoring: LowLevelRefactoring):
     if USE_PROCESS_AND_AUTHORSHIP_METRICS and not refactoring.refactoring_level() == 'class':
         x = x.drop(["authorOwnership", "bugFixCount", "linesAdded", "linesDeleted", "qtyMajorAuthors",
                     "qtyMinorAuthors", "qtyOfAuthors", "qtyOfCommits", "refactoringsInvolved"], axis=1)
+    else:
+        # Remove -1 values from process metrics
```

Comment:
Another suggestion would be to log how many lines are actually removed by this action, so that we can check these numbers later.

Reply:
Done.